Src/CSharpier.Tests/TestFiles/ConstructorDeclaration/ConstructorDeclarations.cst

```diff
@@ -22,11 +22,23 @@ public class BasicClass
 
 public class Initializers : BasicClass
 {
-    public Initializers()
-        : this(true) { }
+    public Initializers() : this(true) { }
 
-    public Initializers(string value)
-        : base(value) { }
+    public Initializers(string value) : base(value) { }
+
+    public Initializers(
+        string reallyLooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooongParameter
+    ) : base(
+        reallyLooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooongParameterIMeanIt
+    ) { }
+
+    public Initializers(
+        string reallyLooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooongParameter
+    ) : base(false) { }
+
+    public ReallyLoooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooogMethodName() : base(
+        false
+    ) { }
```

Comment:
This one should probably change. Here's a real world example.
```
// now formats as
        public DiagnosticLocation(int line, int column) : this(
            $"{DiagnosticProject.DefaultFilePathPrefix}.cs",
            line,
            column
        ) { }
// used to format as this, which I think is preferable.
        public DiagnosticLocation(int line, int column)
            : this($"{DiagnosticProject.DefaultFilePathPrefix}.cs", line, column) { }
```

Reply:
When they are both long, I think it looks good
```
        public ServerInteropTest(
            BrowserFixture browserFixture,
            ToggleExecutionModeServerFixture<Program> serverFixture,
            ITestOutputHelper output
        ) : base(
            browserFixture,
            serverFixture.WithServerExecution().WithAdditionalArguments(GetAdditionalArguments()),
            output
        ) { }
```
Another example with a long base call which wouldn't fit on a single line. It isn't obvious all the parameters are going to a base call.
```
            public TestRemoteRenderer(IServiceProvider serviceProvider, IClientProxy client) : base(
                serviceProvider,
                NullLoggerFactory.Instance,
                new CircuitOptions(),
                new CircuitClientProxy(client, "connection"),
                NullLogger.Instance,
                null
            ) { }
// I'm thinking it should break before : base to make it more clear
            public TestRemoteRenderer(IServiceProvider serviceProvider, IClientProxy client)
                : base(
                    serviceProvider,
                    NullLoggerFactory.Instance,
                    new CircuitOptions(),
                    new CircuitClientProxy(client, "connection"),
                    NullLogger.Instance,
                    null
                ) { }
```

Reply:
Done!